internal/db/db.go

```diff
@@ -34,9 +35,23 @@ func StringToDbType(dialect string) (DbType, error) {
 	}
 }
 
+// DB is a wrapper around the ORM
+type DB struct {
+	*gorm.DB
+}
+
+// SqlDB returns the underlying driver.
```

Comment:
nitpick: I think technically this isn't the "driver" its the `sql.DB` which its [docs](https://pkg.go.dev/database/sql#DB) describe as:

> DB is a database handle representing a pool of zero or more underlying connections.

The drivers are [these packages](https://github.com/golang/go/wiki/SQLDrivers)